### ISSUE TITLE
Missing field `riotIdName`

### ISSUE THREAD

**Issue by PietroM-dev** [MingweiSamuel/Riven#60]
The following function https://docs.rs/riven/latest/riven/endpoints/struct.MatchV5.html#method.get_match from MatchV5 (2.26.0 release) it returns the following error:
```
RiotApiError { reqwest_error: reqwest::Error { kind: Decode, source: Error("missing field `riotIdName`", line: 1, column: 7889) }, retries: 0, response: None, status_code: Some(200) }
```

**Comment by MingweiSamuel** [comment]
What match ID?

**Comment by MingweiSamuel** [comment]
Fixed in https://crates.io/crates/riven/2.27.0

Fixed by 904a6b083fd6b17b87241f14f03b7391db4275ea
